share/models/meta.py

```diff
@@ -1,17 +1,41 @@
+import furl
+import json
+
 from django.db import models
 from django.db import IntegrityError
 from django.contrib.postgres.fields import JSONField
 
 from share.models.base import ShareObject
 from share.models.fields import ShareForeignKey, ShareURLField, ShareManyToManyField
-from share.apps import ShareConfig as share_config
 
 
-__all__ = ('Venue', 'Award', 'Tag', 'Link', 'Subject')
+__all__ = ('Venue', 'Award', 'Tag', 'Subject', 'Identifier', 'Relation')
 
 # TODO Rename this file
 
 
+class Identifier(ShareObject):
+    # https://twitter.com/berniethoughts/
+    url = ShareURLField(unique=True)
+
+    # https://twitter.com/
+    def get_base_url(self):
+        return furl(self.url).origin
+
+
+class Relation(ShareObject):
+    with open('./share/models/relation-types.json') as fobj:
+        # TODO add label to file
+        RELATION_TYPES = [(t['key'], t['uri']) for t in json.load(fobj)]
+
+    subject_work = ShareForeignKey('AbstractCreativeWork', related_name='outgoing_%(class)ss')
```

Comment:
I named them after the "subject-object-predicate" approach Rick [suggested](https://github.com/CenterForOpenScience/SHARE/issues/404#issuecomment-243841316), but you're right, `from` and `to` are probable clearer, considering how overloaded "subject" and "object" already are.